pywbem_mock/_wbemconnection_mock.py

```diff
@@ -3029,11 +3031,11 @@ def _fake_closeenumeration(self, namespace, **params):
             # with the context.
             if context_data['namespace'] != namespace:
                 raise CIMError(CIM_ERR_INVALID_NAMESPACE,
-                               'Incorrect Namespace %s for CloseEnumeration %s '
-                               % (namespace, context_id))
+                               'Incorrect Namespace %r for CloseEnumeration '
```

Comment:
Jse lower case "namespace"

Reply:
Done. Also change 'EnumerationContexts' to 'enumeraton contexts'